Vagrantfile

```diff
@@ -180,7 +181,11 @@ Welcome to the Zulip development environment!  Popular commands:
 Read https://zulip.readthedocs.io/en/latest/testing/testing.html to learn
 how to run individual test suites so that you can get a fast debug cycle.
```

Comment:
Let's move this content to a file `tools/setup/zulip-motd`

Reply:
Moved.